doc/api/index.rst

```diff
@@ -17,35 +17,53 @@ All plotting is handled through the :class:`pygmt.Figure` class and its methods.
 
     Figure
 
-Plotting data and laying out the map:
+Plotting tabular data:
 
 .. autosummary::
     :toctree: generated
 
-    Figure.basemap
-    Figure.coast
-    Figure.colorbar
     Figure.contour
+    Figure.histogram
+    Figure.meca
+    Figure.plot
+    Figure.plot3d
+    Figure.rose
+    Figure.velo
+    Figure.wiggle
+
+Plotting gridded data:
+
+.. autosummary::
+    :toctree: generated
+
     Figure.grdcontour
     Figure.grdimage
     Figure.grdview
-    Figure.histogram
     Figure.image
+
+Plotting map elements:
```

Comment:
Maybe put `basemap` et al. at the top, since the map elements are commonly used.

Reply:
Good suggestion, done in latest commit.